steam-client/src/main/java/com/avenga/steamclient/model/proxy/ProxyState.java

```diff
@@ -0,0 +1,19 @@
+package com.avenga.steamclient.model.proxy;
+
+import lombok.Getter;
+
+import java.net.Proxy;
+
+@Getter
+public class ProxyState {
```

Comment:
its not ThreadSafe. Should it?

Reply:
One Steam client maintain only one connection. These objects are maintained by internal logic, which user can't impact directly. Even if we will use `SteamClient ` in multi-thread logic, rule of the first statement will be true. I will change `connectionFailureCount` to Atomic to be on safe side.